workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -228,8 +235,17 @@ def handle(self, *args, **options):
                         events_for_update[event.slug] = changes
                         print('Detected changes in {}'.format(event.slug))
 
+            except GithubException:
+                print('GitHub error when accessing {} repo'.format(event.slug))
+
+            except socket.timeout:
+                print('Timeout when accessing {} repo'.format(event.slug))
+
             except WrongWorkshopURL:
                 print('Wrong URL for {}'.format(event.slug))
 
             except requests.exceptions.RequestException:
                 print('Network error when accessing {}'.format(event.slug))
+
+            except Exception as e:
+                print('Unknown error ({}): {}'.format(event.slug, e))
```

Comment:
where does this print statement appear? who will see it? what happens afterward? does this halt and catch fire?